.vscode/settings.json

```diff
@@ -1,7 +1,14 @@
 {
   "tailwindCSS.experimental.configFile": {
-    "apps/storybook-react/tailwind.config.js": "packages/design-system-react/src/**",
-    "apps/storybook-react-native/tailwind-intellisense.config.js": "packages/design-system-react-native/src/**"
+    "apps/storybook-react/tailwind.config.js": [
+      "apps/storybook-react/**",
+      "packages/design-system-react/src/**",
+      "packages/design-tokens/stories/**"
+    ],
+    "apps/storybook-react-native/tailwind-intellisense.config.js": [
+      "packages/design-system-react-native/src/**",
+      "apps/storybook-react-native/stories/**"
+    ]
```

Comment:
Tailwind intellisense fixes, allows for autocomplete. Move to separate PR  

<img width="843" alt="Screenshot 2025-05-14 at 9 07 18 PM" src="https://github.com/user-attachments/assets/e5ce6346-d378-4038-8d75-1b3a1f7344ba" />